.github/workflows/CI.yaml

```diff
@@ -13,79 +13,235 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
 
 name: CI
 
 on:
+  pull_request:
   push:
     branches:
       - master
-  pull_request:
-#  schedule: TODO: unpin minor lib versions and check weekly
-#    - 0 0 0 ? * FRI *
+  schedule:
+    - cron: '0 18 * * *'
+
 concurrency:
-  group: ci-it-${{ github.event.pull_request.number || github.ref }}
+  group: CI-plugin-e2e-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
 jobs:
   license-and-lint:
+    # Always check license-and-lint!
```

Comment:
What about forcing this job's success as a prerequisite of all following jobs (now only e2e depends on this) so when this fails we could fail fast?